ports/mimxrt/boards/make-pins.py

```diff
@@ -118,10 +121,17 @@ def print(self):
             self.print_pin_af()
             self.print_pin_adc()
 
+            options = {
+                "LPSR": "PIN_LPSR",
+                "WAKEUP": "PIN_SNVS",
+                "PMIC_ON_REQ": "PIN_SNVS",
+                "PMIC_STBY_REQ": "PIN_SNVS",
+            }
+
             print(
                 "const machine_pin_obj_t pin_{0} = {1}({0}, {2}, {3}, pin_{0}_af, {4}, {5});\n".format(
                     self.name,
-                    "PIN_LPSR" if "LPSR" in self.name else "PIN",
+                    next((options[value] for value in options if value in self.name), "PIN"),
```

Comment:
This line is confusing... maybe `options.get(self.name, "PIN")`, and then make sure all possible LPSR names are included in the `options` dict.

Reply:
> and then make sure all possible LPSR names are included in the `options` dict.

All possible LPSR pads are at least 10, are you sure you want to add them all ?

Reply:
I think it's cleaner to add all LPSR pads.  That way it's explicit what's supported and you'll get an error if you have a typo in a pin name.

Reply:
The MIMXRT1176_af.csv has already the LPSR entries, and the other chips do not seem to have LPSR pads as such. At least I did not find the in the RM.

Reply:
Yes I'll will make the change suggested by @dpgeorge but I noticed it fails to parse the 1176 AF file, will need a small fix.

Reply:
What fails?

Reply:
@robert-hh 
```
line 269, in parse_af_file
    iomux_pin_config[pin.name][af_idx]["inputRegister"].strip("U"),
    ~~~~~~~~~~~~~~~~^^^^^^^^^^
KeyError: 'GPIO_SNVS_00'
```

We need a regex to parse `IOMUXC_GPIO_SNVS_xx` for 117x now that SNVS GPIOs are not skipped, but I think the pad names in the CSV should match the ones in the FSL header.